docs/sweater-comb/index.md

```diff
@@ -1,6 +1,15 @@
 # About this project
 
-COMING SOON:
+## Removing support for old endpoint versions
+
+Removing support for an old API version requires:
+1. Temporarily ignoring the endpoint version code in `.vervet.yaml`
+2. Removing the endpoint versions from `catalog-info.yaml`
```

Comment:
This isn't technically relevant for removing an endpoint, more a by-product of the vervet build process.